src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2383,8 +2383,13 @@ public function get_test_authorization_header() {
 	 * @return array The test result.
 	 */
 	public function get_test_page_cache() {
+		$caching_url  = apply_filters( 'site_status_persistent_page_cache_url', 'homepage' );
 		$description  = '<p>' . __( 'Page cache enhances the speed and performance of your site by saving and serving static pages instead of calling for a page every time a user visits.' ) . '</p>';
-		$description .= '<p>' . __( 'Page cache is detected by looking for an active page cache plugin as well as making three requests to the homepage and looking for one or more of the following HTTP client caching response headers:' ) . '</p>';
+		$description .= '<p>' . sprintf(
+			/* translators: %1$s is the URL or homepage being checked for caching status. */
+			__( 'Page cache is detected by looking for an active page cache plugin as well as making three requests to the %1$s and looking for one or more of the following HTTP client caching response headers:' ),
```

Comment:
If there's only 1 placeholder in a string, it's sufficient to use `%s`. `%1$s` is used when there are multiple placeholders.